src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int dimensionLimit = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figures figure = getRandomFigureFromEnum();
+        switch (figure) {
+            case CIRCLE:
+                Circle circle = new Circle();
+                circle.setRadius(getRandomDimension());

```

Comment:
Let's make a changes that I already mentioned in the type of figures classes and correct this code according to them